README.md

```diff
@@ -1,73 +1,25 @@
-# Greenpeace Planet4 Master Theme
+# Greenpeace Planet 4 Master Theme
```

Comment:
Not a big deal I think, but I noticed that you corrected this here and not in other places (for instance in `CONTRIBUTING.md` or even in this same file below), maybe it would be good to make it consistent?

Reply:
I'm 2 minutes late :sob:

Reply:
Thanks :grin: